### PR TITLE
Add rule of reversible migration

### PR DESCRIPTION
**Pull request by sue445**
Related from https://github.com/bbatsov/rubocop/pull/3854

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -951,6 +951,45 @@ when you need to retrieve a single record by some attributes.
   end
   ```
 
+* <a name="reversible-migration"></a>
+  Migration commands in the `change` method should write only reversible.
+  Reversible migration commands are below.
+  http://api.rubyonrails.org/classes/ActiveRecord/Migration/CommandRecorder.html
```
Comment (bbatsov) [inline README.md]: Make this reference a markdown link, please.

File: README.md [hunk 2]
```diff
@@ -951,6 +951,45 @@ when you need to retrieve a single record by some attributes.
   end
   ```
 
+* <a name="reversible-migration"></a>
+  Migration commands in the `change` method should write only reversible.
+  Reversible migration commands are below.
```
Comment (bbatsov) [inline README.md]: are listed below

File: README.md [hunk 3]
```diff
@@ -951,6 +951,45 @@ when you need to retrieve a single record by some attributes.
   end
   ```
 
+* <a name="reversible-migration"></a>
+  Migration commands in the `change` method should write only reversible.
```
Comment (bbatsov) [inline README.md]: Don't use non-reversible migration commands in the `change` method. (sounds better this way)

**Comment by sue445** [comment]
@bbatsov I fixed and pushed. Sorry, I'm not good at English. 😅

**Comment by sue445** [comment]
thx!

**Pull request merged**